parity/configuration.rs

```diff
@@ -128,6 +128,7 @@ impl Configuration {
 			Some(true) if pruning == Pruning::Specific(Algorithm::Archive) => writeln!(&mut stderr(), "Warning: Warp Sync is disabled because pruning mode is set to archive").expect("Error writing to stderr"),
 			_ => {},
 		};
+		let public_node = self.args.flag_public_node;
```

Comment:
Public node should have only subset of APIs enabled:
`eth,net,parity,rpc,web3`

Some of the disabled APIs are already useless because `AccountProvider` is missing, but stuff like `parity_stopNetwork` should also be disabled.